packages/semantic-release-clean-package-json/src/utils/get-pkg.ts

```
@@ -0,0 +1,63 @@
+// eslint-disable-next-line unicorn/prevent-abbreviations
+import { resolve } from "@visulima/path";
+import AggregateError from "aggregate-error";
```

Biome (1.9.4)

[error] Do not shadow the global "AggregateError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)